README.ja.md

```diff
@@ -112,6 +112,7 @@ $ clive start
 
 - [`init`](#clive-init) - 設定ファイルを作成します。
 - [`start`](#clive-start) - 設定ファイルを読み込み、 cLive を開始します。
+- [`validate`](#clive-validate) - 設定ファイルの構文を検証します。
```

Comment:
Followed this line.
https://github.com/koki-develop/clive/blob/d40f6db1a54648e7c91b2fec56535ce1fd2c00f4/README.md?plain=1#L117C1-L117C1